certified-connectors/DocTOPDFConnector/apiDefinition.swagger.json

```diff
@@ -0,0 +1,107 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "1.6",
+        "title": "Doc To Pdf",
```

Comment:
@yispot Since it is a generic term, can the connector title be "Doc to PDF by Spot solutions" or "Doc to PDF by Spot"

Reply:
changed to 'Doc to PDF by Spot'